lametro/templates/lametro/blocks/article.html

```diff
@@ -0,0 +1,19 @@
+{% load wagtailcore_tags %}
+<article class="mb-5{% if block.value.anchor %} scroll-anchor" id="{{ block.value.anchor }}"{% else %}"{% endif %}>
+    {% if block.value.heading %}
+        <h2>{{ block.value.heading }}</h2>
+    {% endif %}
+    {% for content_block in block.value.content %}
+        {% if content_block.block_type == "table" %}
+            {% with block=content_block %}
+                {% include "lametro/blocks/table.html" %}
+            {% endwith %}
+        {% elif content_block.block_type == "responsive_element" %}
+            {% with block=content_block.value.content.0  breakpoints=content_block.value.breakpoints %}
+                {% include "lametro/blocks/table.html" %}
+            {% endwith %}
```

Comment:
This is suboptimal but I want to think more about other responsive elements before spending more time on this implementation.